src/models/WorkQueue.ts

```diff
@@ -1,53 +1,20 @@
+import { IWorkPoolQueue, WorkPoolQueue } from '@/models/WorkPoolQueue'
 import { IWorkQueueFilter } from '@/models/WorkQueueFilter'
 
-export interface IWorkQueue {
-  readonly id: string,
-  created: Date,
-  updated: Date,
-  name: string,
+export interface IWorkQueue extends IWorkPoolQueue {
   filter?: IWorkQueueFilter,
-  description: string | null,
-  isPaused: boolean,
-  concurrencyLimit: number | null,
-  priority: number,
-  workPoolId: string,
-  workPoolName?: string,
-  status: 'ready' | 'paused' | 'not_ready',
 }
 
-// deployments have new editable field called work_queue_name
-// flow runs do as well
-
-export class WorkQueue implements IWorkQueue {
-  public readonly id: string
-  public created: Date
-  public updated: Date
-  public name: string
+/** @deprecated Prefer `WorkPoolQueue` */
+export class WorkQueue extends WorkPoolQueue implements IWorkQueue {
```

Comment:
We can _likely_ remove these old models - WorkQueue, WorkQueueCreate, WorkQueueEdit, WorkQueueFilter (and their respective api responses). 

Doing this "shim" deprecation for now - this PR is already a bit large.